### DISCUSSION TITLE
CubeCell examples: RADIOLIB_ONBOARD_MODULE or RADIOLIB_BUILTIN_MODULE

### DISCUSSION THREAD

**Discussion by davehummel** [jgromes/RadioLib#479] [Q&A]
For each example updated for the Heltech CubeCell, we have:
```
// or using CubeCell
//SX1262 radio = new Module(RADIOLIB_ONBOARD_MODULE);
```
The RADIOLIB_ONBOARD_MODULE identifier doesnt seem to be defined anywhere.  However RADIOLIB_BUILTIN_MODULE is.  Is this possibly a typo?

**Comment by jgromes** [comment]
You're right, that's indeed a typo. Should be fixed now.

**Reply by davehummel** [reply]
Great.  Thanks for taking the time to build and support this awesome library and to make it work with this strange little board.  Looking forward to using it for telemetry in model flight projects.
